src/plugins/intel_cpu/src/nodes/fullyconnected.h

```diff
@@ -79,6 +80,22 @@ class FullyConnected : public Node {
     ExecutorFactoryPtr<FCAttrs, node::FullyConnected> factory;
     ExecutorPtr executor = nullptr;
     std::string errorPrefix;
+
+    // tensor parallel
```

Comment:
Please wrap it up into some "TensorParallelConfig" structure

Reply:
> Please wrap it up into some "TensorParallelConfig" structure

Done. create a new struct for managing the tensor parallel config.